js/jscript.js

```diff
@@ -0,0 +1,45 @@
+$(function(){
+	//make a click function that will run conversion
+	$("#convertF").click(convertFa);
+
+
+	function convertFa(e){
+		e.preventDefault();
+		//grab user input
+		//parse inputted value 
+		//store parse value into a variable
+		var cToF = parseInt($("#celsius").val());
+		//convert this variable to fah
+		var finalTemp = ( (9/5) * cToF + 32);
+		$("#result").text("Fahrenheit:" + finalTemp + " " + "degrees");	
+
+
+		if (finalTemp < 35) {
+			$("body").css("background-image", "url(images/snow.jpg)");
+		} else if (finalTemp < 64) {
+			$("body").css("background-image", "url(images/spring.jpg" , "background-repeat", "no-repeat"); 
```

Comment:
@cliffjadoo when you have more than one css rule you would write it like this 
el.css({"background-image", "url(images/spring.jpg" , "background-repeat", "no-repeat"});
`